CoreUtilities/test/unit/src/au/gov/asd/tac/constellation/utilities/xml/XmlUtilitiesNGTest.java

```diff
@@ -0,0 +1,907 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package au.gov.asd.tac.constellation.utilities.xml;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.InputStream;
+import java.net.URL;
+import java.util.List;
+import java.util.Map;
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.transform.TransformerException;
+import org.apache.commons.io.FileUtils;
+import static org.apache.poi.hssf.usermodel.HeaderFooter.file;
+import static org.testng.Assert.*;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import org.testng.reporters.Files;
+import org.w3c.dom.Document;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+
+/**
+ *
+ * @author mmattner
+ */
+public class XmlUtilitiesNGTest {
+    
+    private static final String OUTPUT_FILE = "testOutputFile.xml";
+    private static final String XML_HDR = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>";
+    
+    public XmlUtilitiesNGTest() {
+    }
+
+    @BeforeClass
+    public static void setUpClass() throws Exception {
+    }
+
+    @AfterClass
+    public static void tearDownClass() throws Exception {
+    }
+
+    @BeforeMethod
+    public void setUpMethod() throws Exception {
+//        }
+    }
+
+    @AfterMethod
+    public void tearDownMethod() throws Exception {
+    }
+
+    /**
+     * Test of newDocument method, of class XmlUtilities. This effectively creates a new empty 
+     * document.
+     */
+    @Test
+    public void testNewDocument() {
+        System.out.println("testNewDocument");
+        XmlUtilities instance = new XmlUtilities();
+       
+        Document expResult = null;
+        try {
+            expResult = DocumentBuilderFactory.newInstance().newDocumentBuilder().newDocument(); 
+        } catch (ParserConfigurationException ex) {
+            fail("Exception thrown creating sample data");
+        }
```

Comment:
I personally prefer throwing the exception rather than catching if all the catch block does is fail the test

Reply:
done